spacegraphcats/rdomset.py

```diff
@@ -2,7 +2,7 @@
 from collections import defaultdict
 from .graph import Graph, DictGraph
 
-from typing import List, Set, Dict, Any, Union
+from typing import List, Set, Dict, Any, Union, DefaultDict
```

Comment:
This is weird. I saw failures on travis because of this. On my machine, I'm using Python 3.6.1.